package.json

```diff
@@ -1,8 +1,16 @@
 {
   "name": "polyfill-service",
+  "version": "0.0.0",
+  "private": true,
   "description": "A polyfill combinator",
-  "main": "lib/index.js",
-  "bin": "bin/polyfill-service",
```

Comment:
Those would move into the sub-manifests.

`main` ist already in [polyfill-library](https://github.com/Financial-Times/polyfill-service/pull/1267/files/f426ca1e594e0c29e1bf55be460ad5bf0d3c2b57#diff-d3a4d0e3a2b26bc7bdf9e9a9aca2e3d3).

`bin` should be added to [polyfill-service](packages/polyfill-service/package.json)